sdcm/utils/remote_logger.py

```diff
@@ -302,6 +312,128 @@ def _logger_cmd(self) -> str:
         return f"{cmd} >> {self._target_log_file} 2>&1"  # pylint: disable=protected-access
 
 
+class K8sClientLogger(LoggerBase):  # pylint: disable=too-many-instance-attributes
+    READ_REQUEST_TIMEOUT = 300
+    RECONNECT_DELAY = 30
+    CHUNK_SIZE = 1024
+
+    def __init__(self, node: "sdcm.cluster_k8s.BasePodContainer", target_log_file: str):
+        """
+        Reads logs from a k8s pod using k8s client API and forwards it to a file.
+
+        It is self-healing and will re-open the stream if it is closed and continue reading logs from place it stopped.
+        """
+        super().__init__(target_log_file)
+        logging.getLogger('parso.python.diff').setLevel(logging.WARNING)
```

Comment:
is this cause of using `dateutil.parser` ?

Reply:
I didn't track it, but I think so.